manage.py

```diff
@@ -1189,32 +1181,204 @@ def _add_athena_subparser(subparsers):
+
+    _add_default_athena_args(athena_create_table_parser)
+
+    # Validate the provided schema-override options
+    def _validate_override(val):
+        """Make sure the input is in the format column_name=type"""
+        if not '=' in val:
+            raise athena_create_table_parser.error(
+                'Invalid override expression [{}]. The proper format is '
+                '"column_name=value_type"'.format(val))
+
+    athena_create_table_parser.add_argument(
+        '--schema-override',
         nargs='+',
         help=ARGPARSE_SUPPRESS,
         action=UniqueSetAction,
-        default=set())
+        default=set(),
+        type=_validate_override)
+
+
+def _add_athena_rebuild_subparser(athena_subparsers):
+    """Add the athena rebuild-partitions subparser: manage.py athena rebuild-partitions"""
+    athena_rebuild_usage = 'manage.py athena rebuild-partitions'
+
+    athena_rebuild_desc = ("""
+StreamAlertCLI v{}
+Rebuild the partitions for an Athena table
+
+Command:
+
+    manage.py athena rebuild-partitions      Rebuilds the partitions for an Athena table
+
+Required Arguments:
+
+    -b/--bucket                        The name of the S3 bucket to be used for Athena
+                                         query results.
+    -t/--table-name                    The name of the Athena table to create
+
+Optional Arguments:
+
+    --debug                            Enable debug logger output
+
+Examples:
+
+    manage.py athena rebuild-partitions \\
+      --bucket s3.bucket.name \\
+      --table_name my_athena_table
+
+""".format(version))
+    athena_rebuild_parser = athena_subparsers.add_parser(
+        'rebuild-partitions',
+        description=athena_rebuild_desc,
+        usage=athena_rebuild_usage,
+        formatter_class=RawTextHelpFormatter,
+        help=ARGPARSE_SUPPRESS)
+
+    athena_rebuild_parser.set_defaults(subcommand='rebuild-partitions')
+
+    _add_default_athena_args(athena_rebuild_parser)
+
+
+def _add_athena_drop_all_subparser(athena_subparsers):
+    """Add the athena drop-all-tables subparser: manage.py athena drop-all-tables"""
+    athena_drop_all_usage = 'manage.py athena drop-all-tables'
+
+    athena_drop_all_desc = ("""
+StreamAlertCLI v{}
+Drop all tables from an Athena database
+
+Command:
+
+    manage.py athena drop-all-tables     Drop all of the tables from the database
+
+Optional Arguments:
+
+    --debug                              Enable debug logger output
+
+""".format(version))
+    athena_drop_all_parser = athena_subparsers.add_parser(
+        'drop-all-tables',
+        description=athena_drop_all_desc,
+        usage=athena_drop_all_usage,
+        formatter_class=RawTextHelpFormatter,
+        help=ARGPARSE_SUPPRESS)
+
+    athena_drop_all_parser.set_defaults(subcommand='drop-all-tables')
+
+    athena_drop_all_parser.add_argument(
+        '--debug',
+        action='store_true',
+        help=ARGPARSE_SUPPRESS)
+
+
+def _add_default_athena_args(athena_parser):
```

Comment:
I can rename but chose this since the arguments are the 'defaults' related to the actual argparsers that we're constructing here (not the 'defaults' the user will have to provide). Also, you'll notice that not ever argument that is added within the function is in fact 'required' - see the 'debug' param.